modules/site/public/meta.json

```diff
@@ -49,6 +49,32 @@
   },
   {
+    "name": "Pagination",
```

Comment:
It's not transparent to me why this file should be checked in to begin with... seems like it could just be generated and published to `npm` during that process, but @justindujardin might have reasons.